Dockerfile-dind-rootless.template

```diff
@@ -3,7 +3,8 @@ FROM docker:{{ env.version }}-dind
 
 # busybox "ip" is insufficient:
 #   [rootlesskit:child ] error: executing [[ip tuntap add name tap0 mode tap] [ip link set tap0 address 02:50:00:00:00:01]]: exit status 1
-RUN apk add --no-cache iproute2 fuse-overlayfs
+# slirp4netns can be selected as rootlesskit's net driver using "-e DOCKERD_ROOTLESS_ROOTLESSKIT_NET=slirp4netns"
+RUN apk add --no-cache iproute2 fuse-overlayfs slirp4netns
```

Comment:
The default can be changed here:
https://github.com/docker-library/docker/blob/89f7fcb459efa9cb3d19e33f50c3c2d51d4579c1/dockerd-entrypoint.sh#L221

Reply:
But probably we should rather just rely on https://github.com/moby/moby/blob/master/contrib/dockerd-rootless.sh to detect the default